packages/lab/src/__tests__/__e2e__/tabs-next/TabsNext.cy.tsx

```diff
@@ -0,0 +1,68 @@
+import { composeStories } from "@storybook/testing-react";
+import * as tabsStories from "@stories/tabs-next/tabs-next.stories";
+
+const { Default: DefaultTabs } = composeStories(tabsStories);
+
+describe("Given a Tabs", () => {
+  describe("WHEN uncontrolled", () => {
+    describe("WHEN no defaultActiveTabIndex is provided", () => {
+      it("THEN first tab is selected", () => {
+        cy.mount(<DefaultTabs width={400} />);
+        cy.findAllByRole("tab")
+          .eq(0)
+          .should("have.attr", "aria-selected", "true");
+        cy.findByText("Content for Home tab");
+      });
+    });
+    describe("WHEN a defaultActiveTabIndex is provided", () => {
+      it("THEN the defaultActiveTabIndex is selected", () => {
+        cy.mount(<DefaultTabs defaultActiveTabIndex={1} width={400} />);
+        cy.findAllByRole("tab")
+          .eq(1)
+          .should("have.attr", "aria-selected", "true");
+        cy.findByText("Content for Transactions tab");
+      });
+    });
+    describe("WHEN defaultActiveTabIndex is null", () => {
+      it("THEN first tab is selected", () => {
+        // @ts-expect-error
```

Comment:
i believe there is no need for this anymore

Reply:
We do, because this is testing an "incorrect" use case